apps/graphql-playground/src/components/GqlPlayground.tsx

```diff
@@ -87,10 +90,36 @@ function GqlPlayground(props: GqlPlaygroundProps) {
 
   const settings = { 'editor.theme': 'light' };
 
+  useEffect(() => {
+    const unsubscribe = store.subscribe(() => {
+      const query: string = getQuery(store.getState());
+      const result = query.toLowerCase().includes('collection');
+      if (hasCollection !== result) {
+        setHasCollection(result);
+      }
+    });
+    return unsubscribe;
+  }, [hasCollection]);
+
   return (
-    <Provider store={store}>
-      <Playground tabs={tabs} settings={settings} fixedEndpoint={true} {...tabConfig} />
-    </Provider>
+    <>
+      {hasCollection && (
+        <Note>
+          Did you know that the default limit for a collection is <b>100</b>, lowering this would
+          decrease your query complexity! [
+          <a
+            target="_blank"
+            rel="noreferrer"
+            href="https://www.contentful.com/developers/docs/references/graphql/#/reference/collection-fields/arguments:~:text=default%20is%200-,limit,to%20fetch.%20The%20default%20is%20100%20and%20the%20maximum%20is%201000,-where">
```

Comment:
That link should be shortened to `https://www.contentful.com/developers/docs/references/graphql/#/reference/collection-fields/arguments`